scripts/zip_gcs_files.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash -ex
+#
+# Zips up the specified GCS files into a new GCS zip archive.
+#
+# Typical usage:
+#
+# analysis-runner --dataset DATASET --description 'Zip some files' \
+#    --access-level standard --output-dir unused --storage 10G \
+#    scripts/zip_gcs_files.sh gs://BUCKET/ZIPPATH gs://BUCKET/FILE...
+#
+# This will create gs://BUCKET/ZIPPATH containing the other specified
+# files. The script will need storage space for the final zip archive
+# and one localised input file at a time.
+
+zip_path=$1
+shift
+
+cd "$BATCH_TMPDIR"
+
+zip_local=$(basename "$zip_path")
+
+for file_path
```

Comment:
Comment here regarding where file_path is coming from would be nice :-)

Reply:
Fair enough. I made it explicit that it's looping over `"$@"`, i.e., the remaining command-line arguments.